docs/index.rst

```diff
@@ -1,17 +1,67 @@
 .. include:: ../README.rst
    :end-before: About
 
+
 .. include:: ../README.rst
-   :start-after: -----
+   :start-line: 25
    :end-before: Features
 
-.. toctree::
-   :numbered:
 
-   introduction
-   cli
-   restapi
-   contributing
-   changes
-   license
-   authors
+.. include:: ../README.rst
+   :start-line: 32
+   :end-before: Useful links
+
+**Contents**
```

Comment:
Consider killing Contents, to keep up with 'single-autogen-page-and-everything-important-in-docs-reana-io?

Reply:
Do you think it's better to kill it? I find it useful to navigate among sections, even if it's single page.

Reply:
Yeah, in the dictdiffer style.  If the top of the page is short regarding Installation and Usage, it would sort of "drive people away" towards docs.reana.io, which is actually a good thing. The users will know more easily that this page is just an auto-generated reference, that probably only a minority would be interested in. That kind of impression.  We can see how it all looks like once